custom_components/powercalc/data/lidl/HG08008/model.json

```diff
@@ -0,0 +1,13 @@
+{
+    "calculation_strategy": "lut",
+    "measure_description": "Measured with utils/measure script",
+    "measure_device": "Shelly Plug S",
+    "measure_method": "script",
+    "measure_settings": {
+        "SAMPLE_COUNT": 2,
+        "SLEEP_TIME": 3,
+        "VERSION": "v1.3.5:docker"
+    },
+    "name": "HG08008",
```

Comment:
Please put in the full marketed name of the light model here, not the id.

Reply:
I was wondering if this was OK, but it's difficult to find a good name to put here. The `HG08008` is the model name that comes up in [Zigbee2MQTT](https://www.zigbee2mqtt.io/devices/HG08008.html).
On Lidl's webshop where I bought this, they sell this light as (translated) 'Livarno home LED ceiling light smart home' ([link](https://www.lidl.be/p/nl-BE/livarno-home-ledplafondlamp-smart-home/p100348295)), however, they sell in total 7 ceiling lights with the exact same name ([link](https://www.lidl.be/q/nl-BE/search?q=LIVARNO+home+Ledplafondlamp+Smart+Home&variant=c)).

On the box it says 'Smart LED ceiling light', but that also sounds very generic, and I have a feeling the other lights Lidl sells will say the same on the box.

So if I give this measurement either name of Lidl's web shop, or what's on the box, other users will not be able to see which exact model it is. Note that there is already a `HG08010` light in the measurements data ([link](https://github.com/bramstroker/homeassistant-powercalc/tree/master/custom_components/powercalc/data/lidl/HG08010)).

Reply:
I see, thanks for the explanation. It's fine like this.